gr-analog/grc/analog_probe_avg_mag_sqrd_x.xml

```diff
@@ -43,6 +57,12 @@
 		<name>in</name>
 		<type>$type.input</type>
 	</sink>
+	<source>
+		<name>out</name>
+		<type>$type.output</type>
+		<optional>$type.optional</optional>
+		<hide>$type.hide</hide>
```

Comment:
One could use sth like `<hide>#if $type == 'cf' then 'none' else 'all'#</hide>
(same for opt)